pandas/core/series.py

```diff
@@ -2024,6 +2025,7 @@ def drop_duplicates(self, *, inplace: Literal[True]) -> None:
     def drop_duplicates(self, keep=..., inplace: bool = ...) -> Series | None:
         ...
 
+    @deprecate_nonkeyword_arguments(version="2.0", allowed_args=["self"])
```

Comment:
let's allow `"subset"` as well, as it's really common to call that one positionally (else you'll have to update loads of tests)

Reply:
I have added "subset" to the allowed arguments for frame and changed the test in my latest commit to reflect this.